src/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -239,11 +239,12 @@ export const VisibilityCard: React.FC<VisibilityCardProps> = props => {
               value={isAvailable}
               onChange={e => {
                 const { value } = e.target;
-                if (!value) {
-                  onChange({
-                    target: { name: "availableForPurchase", value: null }
-                  });
-                }
+                onChange({
+                  target: {
+                    name: "availableForPurchase",
+                    value: value ? localizeDate(new Date().toString()) : null
```

Comment:
`productSetAvailabilityForPurchase` mutation in the response returns the product with `availableForPurchase` set by the backend. The current logic assumes that we set here the current date if `null` was passed as `startDate` in the mutation. If we change this logic in the backend, `new Date()` here will be invalid, won't it? Can we use the date returned by API here?

Reply:
yes we can, I wanted to avoid refetching all product data only for such small thing